src/base/bittorrent/torrentinfo.h

```diff
@@ -48,14 +47,17 @@ namespace BitTorrent
     class InfoHash;
     struct TrackerEntry;
 
-    class TorrentInfo final : public AbstractFileStorage
+    class TorrentInfo final
     {
         Q_DECLARE_TR_FUNCTIONS(TorrentInfo)
 
     public:
-        explicit TorrentInfo(std::shared_ptr<const lt::torrent_info> nativeInfo = {});
+        TorrentInfo() = default;
         TorrentInfo(const TorrentInfo &other);
 
+        // internal
```

Comment:
sorry, it is not clear what it means, should the following constructor moved to `private:`?